packages/uni_app/android/app/build.gradle

```
@@ -76,6 +76,9 @@ android {
                 "proguard-rules.pro"
             )
         }
+        debug {
+            applicationIdSuffix ".debug"
```

```suggestion
            applicationIdSuffix ".dev"
```
Fits the current name we have in plausible for dev builds.